### DISCUSSION TITLE
Server Error When Uploading Image to Etsy API

### DISCUSSION THREAD

**Discussion by syedappinventiv** [etsy/open-api#1283] [Q&A]
I encountered an issue while attempting to upload an image to the following API endpoint:

https://openapi.etsy.com/v3/application/shops/{shop_id}/listings/{listing_id}/images

The image I am using is:

![1b7ad0ac-0735-4947-9658-b943e684b0d9](https://github.com/user-attachments/assets/a4d01749-ba80-48fc-a393-e249ec5aff52)

However, I am receiving the following response:

`
{
  "error": "Server Error"
}
`
Could anyone help identify the cause of this error or provide guidance on how to resolve it? Thank you in advance for your assistance.

**Comment by turboturtleD** [comment]
Are you using Node fetch?  If so, don't set "multipart/form-data" body schema in the header (don't set any body schema type), let the server figure it out.

Otherwise will probably need to post more information about your request to get community assistance.

**Comment by cyrilchandelier** [comment]
Your image, although it says is a PNG, is a WEBP image. Change the extension to .webp, and it will not trigger the internal server error. That said, it does not look like WEBP is supported by Etsy.

**Reply by showtroylove** [reply]
You can oftentimes rename .webp images to .png and avoid any complaints.

**Comment by showtroylove** [comment]
- Do not offer any more info than required.  
- Do not use both image_id and image.   
- Do not specify a mime-type - it doesn't like that at all.

**Comment by showtroylove** [comment]
You should consider giving credit to the folks (or person) who help your issue.  We're volunteer our time to help.
